container/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM php:7.4-apache
+
+RUN apt update && apt install -y git zip \
```

Docker recommends using apt-get over apt for automated tasks, as its API is less likely to break over time:

```suggestion
RUN apt-get update && apt-get install -y git zip \
```